platformio.ini

```diff
@@ -14,5 +14,5 @@ board = lolin_d32_pro
 framework = arduino
 
 ; If Platformio gives a upload error, please specify this for your respective COM port
-;upload_port = COM[7]
+upload_port = COM[3]
```

Comment:
add this file to the gitignore, the COM value is not the same for every pc. Otherwise you keep pushing this change every time. If changes in the file are ignored, everybody can set the correct COM locally.

Reply:
Done!

Reply:
Is a correct warning thrown to indicate this when a user forgets to set this correctly?